### PR TITLE
feat: Add support for regex type questions #38

### PR DESCRIPTION
**Pull request by AnneliKlamas**

### CODE REVIEW

File: backend/src/main/java/com/quiz/converter/services/MoodleXmlCreatorService.java [hunk 1]
```diff
@@ -65,10 +64,17 @@ private static void addQuestion(Question question, Document doc, Element rootEle
     }
 
     private static void addQuestionOptions(Question question, Document doc, Element questionElem) {
-        var shuffleTagElem = doc.createElement("shuffleanswers");
-        var shuffleText = question.shuffle() ? "1" : "0";
-        shuffleTagElem.setTextContent(shuffleText);
-        questionElem.appendChild(shuffleTagElem);
+        if (question.type().equals(QuestionType.REGULAR_EXPRESSION)) {
```
Comment (TobiasReiter) [inline backend/src/main/java/com/quiz/converter/services/MoodleXmlCreatorService.java:67]: Also TEXT_MATCH?

Comment (TobiasReiter) [reply backend/src/main/java/com/quiz/converter/services/MoodleXmlCreatorService.java:67]: .

**Review by TobiasReiter — APPROVED** [review]

**Review by TobiasReiter — COMMENTED** [review]
.

File: backend/src/main/java/com/quiz/converter/services/MoodleXmlCreatorService.java (hunk 1 above)
Comment (AnneliKlamas) [reply backend/src/main/java/com/quiz/converter/services/MoodleXmlCreatorService.java:66]: Text match doesn't need "studentshowalternate" element for it to work.

**Pull request merged**